scripts/datastore_upgrade.py

```diff
@@ -0,0 +1,422 @@
+import json
+import logging
+import os
+import subprocess
+import sys
+import time
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '../lib'))
+import appscale_info
+from constants import CONTROLLER_SERVICE
+from constants import APPSCALE_HOME
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '../AppDB'))
+import appscale_datastore_batch
+import datastore_server
+import entity_utils
+from dbconstants import *
+from zkappscale import zktransaction as zk
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "../AppServer"))
+from google.appengine.api import datastore_errors
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "../InfrastructureManager"))
+from utils import utils
+
+# Default ZooKeeper port formatted with a colon.
+ZK_PORT_WITH_COLON = ":" + str(zk.DEFAULT_PORT)
```

Comment:
Please keep the port separate from ":" and format accordingly below. It's cleaner imho.